aspnetcore/host-and-deploy/linux-apache.md

```diff
@@ -255,6 +255,21 @@ For time filtering, specify time options with the command. For example, use `--s
 sudo journalctl -fu kestrel-hellomvc.service --since "2016-10-18" --until "2016-10-18 04:00"
 ```
 
+## Data protection
+
+The [ASP.NET Core Data Protection stack](xref:security/data-protection/index) is used by several ASP.NET Core [middlewares](xref:fundamentals/middleware/index), including middleware used in authentication. Even if Data Protection APIs aren't called by user code, data protection should be configured to create a persistent cryptographic [key store](xref:security/data-protection/implementation/key-management). If data protection isn't configured, the keys are held in memory and discarded when the app restarts.
```

Comment:
Needs to explain why? Mention its used by, for example, cookie middleware and CSRF protections.